### DISCUSSION TITLE
Community Standards Checklist incomplete when All users selected in Report content setting

### DISCUSSION THREAD

**Discussion by cpswan** [community/community#59546] [Repositories]
### Select Topic Area

Bug

### Body

As I've been checking the Community Standards Checklist for some new repos I've noticed that the `Repository admins accept content reports` only gets a tick:

![image](https://github.com/community/community/assets/478926/4efe4d17-b0ff-479f-8152-d23cc639d792)

When Report content setting is at `Prior contributors and collaborators`:

![image](https://github.com/community/community/assets/478926/dcfb7b11-bef4-4e7e-9297-feca872a964a)

If it's set to the more open `All users`:

![image](https://github.com/community/community/assets/478926/e2cd3c4e-169d-4d9c-94f3-e6c541fbfd57)

Then the tick disappears from the checklist (and the `Enable` button is back):

![image](https://github.com/community/community/assets/478926/bc1de1e9-3498-4bbd-b358-86c1e8977c6c)

This seems like a bug, as surely `Repository admins accept content reports` is (more) true for `All users` as it is for  `Prior contributors and collaborators`.

**Comment by klieret** [comment]
Still happening in July 2025

**Comment by adamretter** [comment]
Still happening in September 2025

**Comment by Kichura** [comment]
Can confirm here aswell, not exactly sure why no GitHub staff is looking into this one in terms of the community standards checklist.

**Comment by Nugine** [comment]
Still happening in 2026-02
